src/pages/docs/liveobjects/rest-api-usage.mdx

```diff
@@ -400,163 +248,46 @@ The response includes the object tree starting from the specified object ID. Nes
-  },
-  "siteTimeserials": {
-    "e02": "01745828596522-000@e025VxXLABoR0C19591332:001"
-  },
-  "tombstone": false
-}
-```
+  ```
 </Code>
 
-#### Pagination <a id="fetching-objects-get-pagination"/>
 
-When fetching objects in a tree structure, the response can be paginated using the `limit` query parameter to specify the maximum number of objects to include in the response. If a nested child object exists which cannot be included in the response because the limit has been reached, it will be included by reference to its object ID rather than its value:
+The `path` param can also be provided here to return a subset of the object tree starting from the specified object ID.
 
 <Code>
-```shell
-  curl "https://main.realtime.ably.net/channels/my-channel/objects/root?children=true&limit=1" \
-    -u {{API_KEY}}
-```
+  ```shell
+  curl "https://main.realtime.ably.net/channels/my-channel/object/map:fNDMOPlPriqid9SUitqIuP_S-6u5YUYQzxjs39IM68k@1760448542413?path=votes" \
+  -u {{API_KEY}}
+  ```
 </Code>
 
 <Code>
-```json
-{
-  "objectId": "root",
-  "map": {
-    "entries": {
-      "votes": {
-        "data": {
-          "objectId": "map:ja7cjMUib2LmJKTRdoGAG9pbBYCnkMObAVpmojCOmek@1745828596519"
-        }
-      }
-    }
+  ```json
+  {
+    "down": 5,
+    "up": 10
   }
-}
-```
-</Code>
+  ```
 
-To obtain the next page, make a subsequent query specifying the referenced object ID as the entrypoint:
+  This request is billed per object returned.
 
-<Code>
-```shell
-  curl "https://main.realtime.ably.net/channels/my-channel/objects/map:ja7cjMUib2LmJKTRdoGAG9pbBYCnkMObAVpmojCOmek@1745828596519?children=true&limit=1" \
-    -u {{API_KEY}}
-```
-</Code>
-
-<Code>
-```json
-{
-  "objectId": "map:ja7cjMUib2LmJKTRdoGAG9pbBYCnkMObAVpmojCOmek@1745828596519",
-  "map": {
-    "entries": {
-      "down": {
-        "data": {
-          "objectId": "counter:JbZYiHnw0ORAyzzLSQahVik31iBDL_ehJNpTEF3qwg8@1745828651669"
-        }
-      },
-      "up": {
-        "data": {
-          "objectId": "counter:iVji62_MW_j4dShuJbr2fmsP2D8MyCs6tFqON9-xAkc@1745828645269"
-        }
-      }
-    }
-  }
-}
-```
 </Code>
 
-#### Cyclic references <a id="fetching-objects-get-cycles"/>
+### Cyclic references <a id="fetching-objects-get-cycles"/>
 
-When using the `children` query parameter, cyclic references in the object tree will be included as a reference to the object ID rather than including the same object in the response more than once.
+For both the full object and the compact response formats cyclic references in the object tree will be included as a
+reference to the object ID rather than including the same object in the response more than once.
 
-For example, if we created a cycle in the object tree by adding a reference to the root object in the `votes` `LiveMap` instance with the following operation:
+For example, if we created a cycle in the object tree by adding a reference to the root object in the `poll` `LiveMap` instance with the following operation:
```

Comment:
We want to avoid referencing the "root" object from now on, so I think we need to rework/reword this example